src/main/java/com/redhat/devtools/lsp4ij/LanguageServiceAccessor.java

```diff
@@ -584,4 +589,56 @@ private void disposeAllServers() {
         }
     }
 
+    /**
+     * Increments the language server definition's modification tracker.
+     */
+    void incrementModificationCount() {

```

Comment:
Package-protected method to increment the accessor-level modification tracker when any of the project's servers changes status.